lib/galaxy/webapps/galaxy/api/tool_dependencies.py

```diff
@@ -193,12 +193,20 @@ def summarize_toolbox(self, trans: ProvidesAppContext, **kwds):
         """
         GET /api/dependency_resolvers/toolbox
 
-        Summarize requirements across toolbox (for Tool Management grid). This is an experiemental
-        API particularly tied to the GUI - expect breaking changes until this notice is removed.
+        Summarize requirements and their dependencies across toolbox (for Tool Management grid).
+        This is an experimental API particularly tied to the GUI - expect
+        breaking changes until this notice is removed.
 
         Container resolution via this API is especially experimental and the container resolution
         API should be used to summarize this information instead in most cases.
 
+        The results are computed with `_requirements_to_dependencies_dict()` of
+        `tool_util.deps.DependencyManager` Note that for container resolution by
+        default resolvers are skipped that require online access or may
+        build/cache a container, i.e.  resolvers that
+        - define `builds_on_resolution` i.e. `build_mulled` and `build_mulled_singularity`
+        - unless `install=True` only resolvers starting with `"cached"`, `"explicit"`, or `"fallback"` are used
```

Comment:
I don't think these details belong in the route docstring.

Reply:
I tried to remove some of the details. Better?